### PR TITLE
Cleaned up test for clean_inchis

### LINKED ISSUE #99: don't load data from file in unit test
(copied from https://github.com/matchms/matchms/pull/73#pullrequestreview-388954200)

data file should be single spectrum since it supposed to bring to light a single error. Ideally, the datafile has only the minimum set of properties to be useful. We may not even need a data file, we could instantiate a Spectrum for this (fewer moving parts is better). For example like I did here https://github.com/matchms/matchms/blob/9fb639f43f0cf1fa242e2e5cdf0eb5e5efa0f7e7/tests/test_normalize_intensities.py#L8-L16

### LINKED ISSUE #101: refactor / check tests/test_clean_inchis.py
https://github.com/matchms/matchms/blob/10cd8913fcdce04f9c5447aed6eef5677a775c44/tests/test_clean_inchis.py#L8-L29

### PR DESCRIPTION
**Pull request by florian-huber**
As suggested by @jspaaks in #101 I removed the loading of the mgf file and replaced it by synthetic spectra.
(I also did a minor change to the print output of the clean_inchis function)

This was the last unit test loading the mgf file. So this PR should close #99 and #101.

### CODE REVIEW

File: tests/test_clean_inchis.py [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
-    assert query_spectrum_raw.get("inchi").startswith('InChI='), 'expected different InChI'
-    assert query_spectrum.get("inchi").startswith('"InChI='), 'InChI style not as expected.'
-    original_inchi = reference_spectrums_raw[2].get("inchi")
-    assert original_inchi.startswith('"InChI=CCCCCCCCCCCCCCCC(=O)'), "expected misplaced smiles"
-    modified_inchi = reference_spectrums[2].get("inchi")
-    assert modified_inchi.startswith('"InChI=1S/C24H50NO7P/'), "inchi was not converted correctly"
+    spectrum1 = clean_inchis(spectrum_in1)
+    spectrum2 = clean_inchis(spectrum_in2)
+    assert spectrum1.get("inchi").startswith('"InChI='), "InChI style not as expected"
+    assert spectrum2.get("inchi").startswith('"InChI='), "InChI style not as expected"
```
Comment (jspaaks) [inline tests/test_clean_inchis.py]: I expected an ``assert spectrum1 == spectrum2`` here, to verify the harmonization. Alternatively, you could make two unit tests, one for everything related to ``spectrum_in1``, the other for ``spectrum_in2``.

**Review by jspaaks — CHANGES_REQUESTED** [review]
Nice work just a minor comment.

File: tests/test_clean_inchis.py (hunk 1 above)
Comment (florian-huber) [reply tests/test_clean_inchis.py]: Good suggestion, thanks. I prefer the first option. I hence replaced line 31 with ``assert spectrum1 == spectrum2``.

Comment (jspaaks) [reply tests/test_clean_inchis.py]: nice

**Review by jspaaks — APPROVED** [review]

**Comment by florian-huber** [comment]
Thanks Juriaan for the review!

**Pull request merged**